docker/scripts/install_python_versions

```diff
@@ -20,8 +20,8 @@ if [[ ! -n ${VERSIONS_TO_INSTALL} ]]; then
   exit 1
 fi
 
-# Download requirements.txt file from GitHub
-wget https://raw.githubusercontent.com/h2oai/h2o-3/${H2O_BRANCH}/h2o-py/requirements.txt
+# Download test-requirements.txt file from GitHub
```

Comment:
I think it would be better to keep the `requirements.txt` file and add the extra requirements to `test-requirements.txt`. This way we will also (to some extent) test that our `requirements.txt` have the right dependencies.